lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -126,16 +138,16 @@ define((require, exports, module) ->
         @lastChangeTime = null
       else
         @currentDelayedUpdateId = null
-        { row: cursorRow, column: cursorColumn } = @editor.getCursorPosition()
-        curContext = LatexParsingContext.getContext(@editor.getSession(), cursorRow, cursorColumn)
+        cursorPos = @editor.getCursorPosition()
+        curContext = LatexParsingContext.getContext(@editor.getSession(), cursorPos.row, cursorPos.column)
 
         if curContext == "equation"
           @updateRange()
         else
           @destroyRange()
           @disableUpdates()
 
-        if @rangeExists and @rangeCorrect
+        if @currentRange?
```

Comment:
The logic has changed. Now the preview is displayed, whether the range is correct or not. If it's not correct, the user is shown a proper notification, that something isn't right.